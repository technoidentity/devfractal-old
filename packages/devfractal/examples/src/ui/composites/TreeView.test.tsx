import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { TreeViewStructure } from './TreeView'

test('TreeView Structure', () => {
  const component: ReactTestRenderer = renderer.create(<TreeViewStructure />)
  expect(component.toJSON()).toMatchSnapshot()
})
