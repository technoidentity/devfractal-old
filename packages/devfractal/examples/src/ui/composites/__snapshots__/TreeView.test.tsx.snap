// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TreeView Structure 1`] = `
<div
  className="tree-element"
  style={
    Object {
      "marginLeft": " 4 +'px'",
    }
  }
>
  <span
    className="toggler"
    onClick={[Function]}
  />
  <strong>
      
    Rectangle
    : 
  </strong>
  {
  <div
    className="tree-element"
    style={
      Object {
        "marginLeft": 16,
      }
    }
  >
    <span
      className="toggler"
      onClick={[Function]}
    />
    <strong>
        
      top
      : 
    </strong>
    {
    <p
      className="tree-element"
      style={
        Object {
          "marginLeft": "16 + 'px'",
        }
      }
    >
      <strong>
        x
        : 
      </strong>
      1
      ,
    </p>
    <p
      className="tree-element"
      style={
        Object {
          "marginLeft": "16 + 'px'",
        }
      }
    >
      <strong>
        y
        : 
      </strong>
      2
      
    </p>
    }
    ,
  </div>
  <div
    className="tree-element"
    style={
      Object {
        "marginLeft": 16,
      }
    }
  >
    <span
      className="toggler"
      onClick={[Function]}
    />
    <strong>
        
      bottom
      : 
    </strong>
    {
    <p
      className="tree-element"
      style={
        Object {
          "marginLeft": "16 + 'px'",
        }
      }
    >
      <strong>
        y
        : 
      </strong>
      2
      ,
    </p>
    <p
      className="tree-element"
      style={
        Object {
          "marginLeft": "16 + 'px'",
        }
      }
    >
      <strong>
        z
        : 
      </strong>
      3
      
    </p>
    }
    
  </div>
  }
  
</div>
`;
